tests/all/utils/fixture.rs

```diff
@@ -474,3 +474,207 @@ pub fn wbg_test_node() -> Fixture {
+            r#"
+                extern crate wasm_bindgen;
+                // extern crate project_b;
+                use wasm_bindgen::prelude::*;
+
+                // Import the `window.alert` function from the Web.
+                #[wasm_bindgen]
+                extern {
+                    fn alert(s: &str);
+                }
+
+                // Export a `greet` function from Rust to JavaScript, that alerts a
+                // hello message.
+                #[wasm_bindgen]
+                pub fn greet(name: &str) {
+                    alert(&format!("Hello, {}!", name));
+                }
+            "#,
+        );
+    }
+
+    fn project_b_fixture(fixture: &mut Fixture) {
+        fixture.file(
+            PathBuf::from("project_b/README"),
+            "# Project Beta Fixture\n",
+        );
+        fixture.file(
+            PathBuf::from("project_b/Cargo.toml"),
+            r#"
+            [package]
+            authors = ["The wasm-pack developers"]
+            description = "so awesome rust+wasm package"
+            license = "WTFPL"
+            name = "project_b"
+            repository = "https://github.com/rustwasm/wasm-pack.git"
+            version = "0.1.0"
+
+            [lib]
+            crate-type = ["cdylib"]
+
+            [dependencies]
+            wasm-bindgen = "=0.2.21"
+
+            [dev-dependencies]
+            wasm-bindgen-test = "=0.2.21"
+        "#,
+        );
+        fixture.file(
+            PathBuf::from("project_b/src/lib.rs"),
+            r#"
+                extern crate wasm_bindgen;
+                use wasm_bindgen::prelude::*;
+
+                // Import the `window.alert` function from the Web.
+                #[wasm_bindgen]
+                extern {
+                    fn alert(s: &str);
+                }
+
+                // Export a `greet` function from Rust to JavaScript, that alerts a
+                // hello message.
+                #[wasm_bindgen]
+                pub fn greet(name: &str) {
+                    alert(&format!("Hello, {}!", name));
+                }
+            "#,
+        );
+    }
+
+    let mut fixture = Fixture::new();
+    project_b_fixture(&mut fixture);
+    project_a_fixture(&mut fixture);
+    project_main_fixture(&mut fixture);
+    fixture
+}
+
+pub fn project_with_crc() -> Fixture {
+    let fixture = Fixture::new();
+    fixture.file(PathBuf::from("README"), "# Project Beta Fixture\n");
+    fixture.file(
+        PathBuf::from("Cargo.toml"),
+        r#"
+            [package]
+            authors = ["The wasm-pack developers"]
+            description = "so awesome rust+wasm package"
+            license = "WTFPL"
+            name = "proj"
+            repository = "https://github.com/rustwasm/wasm-pack.git"
+            version = "0.1.0"
+
+            [lib]
+            crate-type = ["cdylib"]
+
+            [dependencies]
+            wasm-bindgen = "=0.2.21"
+            crc = "*"
```

Comment:
this is *extremely* fragile

I'm not sure about this strategy overall

Reply:
yeah i will need to talk a look at this a bit closer but i am not in favor of * deps in general, and i would prefer a more general use case test

Reply:
Yeah I also agree, and I'm glad you both point it out.

In this case, I was having a problem installing `crc = "1.8.1"`. I left it as * because pinning the version makes sense today, but if there was a change tomorrow in crc, we wouldn't catch it.

I can pin the version if that is preferred 😄